src/groups/cover.js

```diff
@@ -62,6 +64,17 @@ module.exports = function (Groups) {
 	};
 
 	Groups.removeCover = async function (data) {
+		const fields = ['cover:url', 'cover:thumb:url'];
+		const values = await Groups.getGroupFields(data.groupName, fields);
+		await Promise.all(fields.map((field) => {
+			if (!values[field] || !values[field].startsWith('/assets/uploads/files/')) {
+				return;
+			}
+			const filename = values[field].split('/').pop();
+			const filePath = path.join(nconf.get('upload_path'), 'files', filename);
+			return file.delete(filePath);
+		}));
+
 		await db.deleteObjectFields(`group:${data.groupName}`, ['cover:url', 'cover:thumb:url', 'cover:position']);
```

Comment:
`[...fields, 'cover:position']`?

Reply:
Oh, forgot it again. :) Seems like this way it results in fewer CPU cycles though. :P